torch_xla/csrc/aten_tensor_ops.cpp

```diff
@@ -17,8 +19,9 @@ at::Tensor& celu_(at::Tensor& self, at::Scalar alpha) {
 }
 
 std::tuple<at::Tensor, at::Tensor, at::Tensor> native_group_norm(
-    const at::Tensor& input, const at::Tensor& weight, const at::Tensor& bias,
-    int64_t N, int64_t C, int64_t HxW, int64_t group, double eps) {
+    const at::Tensor& input, c10::optional<at::Tensor> weight,
```

Comment:
Let's pass optional values by `const&` instead of value ... all over this PR.

Reply:
Ah nice catch about const&! In fact we'll need to update RegistrationDeclarations.h in the pytorch PR first. Will update this PR once it's done!

Reply:
You could decouple the XLA change (where we use `const&`) from the PT change.
They do not need to be necessarily tied together.

Reply:
hmmmm but we check cpp_sig to match RegistrationDeclarations.h in codegen?

Reply:
I mean, the aten_xla_type.* remains (until you change PT side) passed by value ...